### PR TITLE
tests for edit participant endpoint (fixes #214)

### LINKED ISSUE #214: test for edit participant endpoint
preparing pytests for the endpoint "edit participant" in "test_participant_details.py"

### PR DESCRIPTION
**Pull request by kristhina**
to test:
make test

### CODE REVIEW

**Comment by OtisRed** [comment]
@kristhina Travis faild while on the command black --check. Something went wrong with Black then.

File: backend/tests/test_participant_details.py [hunk 1]
```diff
@@ -22,21 +22,54 @@ def test_get_delete_participant_when_logged_in(
     else:
         assert response["message"] == "Participant deleted successfully."
 
+@pytest.mark.parametrize("method", ["put"])
```
Comment (stanislawK) [inline backend/tests/test_participant_details.py]: You could remove that line  - parametrize is useful when you have more arguments - there is one only, and you don't use it in test.

**Comment by w1stler** [comment]
@kristhina please resolve conflicts.

**Comment by kristhina** [comment]
> @kristhina please resolve conflicts.

resolved

**Pull request merged**